src/bin/flamegraph.rs

```diff
@@ -46,6 +48,50 @@ struct Opt {
     #[structopt(long = "cp")]
     cp: bool,
 
+    /// Change title text
+    #[structopt(long = "title", default_value = "Flame Graph")]
```

Comment:
We should use `DEFAULT_TITLE` here too.

Reply:
I think this should work:
```Rust
#[structopt(long = "title", raw(default_value = "DEFAULT_TITLE"))]
```

Reply:
I think you have to use
```rust
#[structopt(long = "title", raw(default_value = DEFAULT_TITLE))]
```

Reply:
It would be nice to have constants for all of the defaults, but the above trick will only work for `&str` constants. I don't know how we would use `f64` constants and others with `structopt`.

Reply:
@jasonrhansen I think we should be able to use `raw` for all of them, no?

Reply:
I just tried using `raw` with an `f64` constant and the compiler complained.

```
error[E0308]: mismatched types
 --> src/bin/flamegraph.rs:9:17
  |
9 | #[derive(Debug, StructOpt)]
  |                 ^^^^^^^^^ expected &str, found f64
  |
  = note: expected type `&str`
             found type `f64`
```

Reply:
Hmm, that seems wrong. Maybe try filing an issue with structopt?

Reply:
I think the problem is that `clap` itself requires a `&str`.
```Rust
pub fn default_value(self, val: &'a str) -> Self
```

Reply:
Oooh, that's neat. I think I'd use a different field name than `name` though, since it's really something like `strval`. I'd expect `name` to be the name of the parameter.

Also keep in mind that `stringify!` doesn't quite do what you'd want, since it makes a string representation of the [tokens of the expression](https://doc.rust-lang.org/std/macro.stringify.html). For example, if you did
```rust
define!(PORT: u32 = 1024 + 16);
```
You'd end up with `default_value = "1024 + 16"`. We could perhaps limit it to take `$val:tt` instead (note the lack of a multiplier), which would only allow a single token, but I don't know if that's any better.

Reply:
IMHO, the solution should not overcomplicate the problem it solves, so it is better to leave this "duplication" as is.

Reply:
I was thinking the same thing about the field being called `name`. `strval` makes more sense. And yes, we'll need to be careful to only use a single literal with the macro, but that's all we need anyway. It could definitely trip someone up though in the future. `$val::tt` would help, but wouldn't it still allow an expression in parentheses like this `(1024 + 16)`?

Reply:
@AnderEnder I actually quite worry about having mismatched defaults. They are _also_ being duplicated in docstrings, and being able to refer to public constants instead would be nice. We should do this as part of a separate PR though :)

@jasonrhansen I'm actually not sure; you might be right. But at least it stops the easiest to commit mistakes..

Reply:
Err, yeah, that doesn't work sadly:
```
error: proc-macro derive panicked
  --> src/bin/flamegraph.rs:11:17
   |
11 | #[derive(Debug, StructOpt)]
   |                 ^^^^^^^^^
   |
   = help: message: invalid structopt syntax: # [ structopt ( long = "title" , raw ( default_value = DEFAULT_TITLE ) ) ]

error: aborting due to previous error
```
Let's leave it for another PR.

Reply:
`DEFAULT_TITLE` needs to be in quotes like this.
```
#[structopt(long = "title", raw(default_value = "DEFAULT_TITLE"))]
```

Reply:
Won't that make the title quite literally "DEFAULT_TITLE"?

Reply:
Not with `raw`.

Reply:
Opened https://github.com/jonhoo/inferno/issues/91